tests/db_tests/test_queries.py

```diff
@@ -44,17 +45,17 @@ def outcome_data_structure_common_test(outcome_data_df):
 
 
 def test_get_download_data_no_filters(seeded_test_client, additional_test_data):
-    assert len(OutcomeData.query.all()) == 31
-    programme_with_no_projects = additional_test_data["programme_with_no_projects"]
-    programme_outcome_child_of_no_projects = additional_test_data["outcome_no_projects"]
-    assert programme_outcome_child_of_no_projects.id in [row.id for row in OutcomeData.query.all()]
+    assert len(OutcomeData.query.all()) == 30
+    # programme_with_no_projects = additional_test_data["programme_with_no_projects"]
+    # programme_outcome_child_of_no_projects = additional_test_data["outcome_no_projects"]
+    # assert programme_outcome_child_of_no_projects.id in [row.id for row in OutcomeData.query.all()]
```

Comment:
Do we want to delete this commented out test code?

If so we can probably also delete programme_with_no_projects from additional_test_data as its not set up to be referenced by the programme junction and I can't see that it's used anywhere else

Will carry on reviewing tomorrow and test the migration locally etc 👍

Reply:
yeah, I should have removed all this. I'll remove and also remove programme_with_no_projects from the conftest data

Reply:
done